qgis_hub_plugin/gui/resource_browser.py

```diff
@@ -221,10 +235,20 @@ def download_resource(self):
         resource = self.selected_resource()
         file_extension = os.path.splitext(resource.file)[1]
 
+        # Set the default download location
+        default_download_location = "~/Downloads"
+
+        # Read the stored download location
+        download_location = read_settings(
+            "myplugin/downloadLocation", default_download_location
```

Comment:
Obviously don't use `myplugin` as the prefix. `QGISResourceHubPlugin` is a better option.

Also put the prefix in the function to read and write, so no need to write it everywhere.